earth_enterprise/rpms/build.gradle

```diff
@@ -43,6 +43,17 @@ apply plugin: "nebula.ospackage"
 
 import org.opengee.shell.GeeCommandLine
 
+// Varables used for specifying location of two files
+// 'gee_long_version.txt' and 'gepgcommon'
+def working_dir = "../src/"
+
+// Use the command line option "-PDEBUGRPMS" in order to build
+// for NATIVE-DBG-x86_64.  Default is NATIVE-REL-x86_64
+def native_x86 = ( project.hasProperty( 'DEBUGRPMS' ) ? "NATIVE-DBG-x86_64/" : "NATIVE-REL-x86_64/" )
+
+// Build the actual file to be used
+def long_txt = working_dir + native_x86 + "gee_long_version.txt"
```

Comment:
Minor: might be nice to have a more descriptive variable name than `long_txt` - even `long_version_txt` would work.

Reply:
Fair point.  I agree.

Reply:
There's already a variable that has the path to `gee_long_version.txt` as a `File` below.  It is called `openGeeVersionFile`.  I think the name should correspond.  Or, just use the `File` variable.  It converts to `String`, and supports many other path operations.

Reply:
Minor: Also, I think it's more readable to use Groovy string templates, rather than string arithmetic to construct a string.

E.g.:

```Groovy
def openGeeVersionFile = new File("../src/NATIVE-${project.buildName}-x86_64/gee_long_version.txt")
```